script/release/uploaders/upload.py

```diff
@@ -228,7 +228,7 @@ def purify_extra_data(mm, offset, length, compressed_size=0):
     ZIP64_EXTRA_HEADER = 0x0001
     zip64_extra_struct = Struct("<HHQQ")
     # ZIP64.
-    # When a ZIP64 extra field is present his 8byte length
+    # When a ZIP64 extra field is present their 8byte length
```

Comment:
This was a spelling mistake.  It should not have been `his` but instead `this`
```suggestion
    # When a ZIP64 extra field is present this 8byte length
```